### DISCUSSION TITLE
seismic wave propagation on a 2D grid

### DISCUSSION THREAD

**Discussion by DanBrox** [geodynamics/sw4#162] [Q&A]
Would like to know if it is possible to simulate seismic wave propagation on a 2D grid in SW4, assuming elastodynamics variables (e.g. stress / strain) are independent of one of the spatial coordinates.  I know this could be done on a 3D grid in SW4, but I'd like to use the spatial symmetry of the solution to reduce the time required to compute solutions.

**Comment by BFF-LEI-Lei** [comment]
I have the same question!
Can't we set one of the dimensions of the grid to zero?
